packages/core/src/sdk/search/useTopSearch.ts

```diff
@@ -41,6 +42,7 @@ function useTopSearch(
   })
 
   return {
+    data,
     terms: (data?.search.suggestions.terms ?? initialTerms).slice(0, limit),
```

Comment:
Same here

Reply:
the same [here](https://github.com/vtex/faststore/pull/2051/commits/97fb5fdc95aabdfbd5a11f51a2aa090435ffa5fe).